src/QsCompiler/Compiler/PluginInterface.cs

```diff
@@ -12,10 +12,17 @@
 namespace Microsoft.Quantum.QsCompiler
 {
     /// <summary>
-    /// Lists the priorities for built-in rewrite steps.
+    /// Lists the priorities for built-in rewrite steps. Steps with a larger priority number have higher priority and
+    /// will be executed first.
     /// </summary>
     public static class RewriteStepPriorities
     {
+        /// <summary>
+        /// Priority of the built-in transformation that infers the minimum runtime capabilities required by each
+        /// callable.
+        /// </summary>
+        public const int CapabilityInference = 1200;
```

Comment:
It was originally last, but I think I moved it up because it was not possible to run capability inference on generated code due to bug #634. Since that bug has been fixed, I'll try moving it back again.